bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -388,73 +388,81 @@ var Tabzilla = (function (Tabzilla) {
             return false;
         }
 
+        var normalize = function (lang) {
```

Comment:
Worth preserving the comment here? `// Normalize the user language in the form of ab or ab-CD`